src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -411,7 +411,7 @@ private IntPtr marshalHeaders(IEnumerable<IHeader> headers)
 
                 return errorCode;
             }
-            catch 
+            catch
```

Comment:
These whitespace fixes were done automatically by the vscode formatter. @mhowlett Let me know if I need to revert them.

Reply:
There are lots and lots of places where there's trailing whitespace in this repository. I think we should standardize on using something like dotnet format, at least for the whitespace. Would be a big step to do the first time, though.

Reply:
yeah, annoying for git blame, but i agree.

Reply:
I will take a look if we can integrate some .NET checkstyle in the repo so that automatically on committing code can format according to that.